README.md

```diff
@@ -96,6 +96,7 @@ Seznam je určený pro všechny frontend designery, kteří se ptají, kde je mo
 - [Atom](https://atom.io/)
 - [Sublime Text](https://www.sublimetext.com/)
 - [Visual Studio Code](https://code.visualstudio.com/)
+- [Webstorm](https://confluence.jetbrains.com/display/WI/WebStorm+EAP)
```

Comment:
@jetpack3331 díky za update. Nebylo by lepší přidat odkaz na Webstorm vedoucí na oficiální stránku https://www.jetbrains.com/webstorm/? Přeci jen uvedený odkaz na Early Access Program je jen pro vyvolené, které mají zájem odebírat nejnovější verze (kolikrát ještě méně stabilnější).

Reply:
Jo to by asi bylo, ja jsem byl jen zvykly si stahovat ten EAP :)